metagpt/provider/openai_api.py

```diff
@@ -323,3 +325,29 @@ async def amoderation(self, content: Union[str, list[str]]):
     async def _amoderation(self, content: Union[str, list[str]]):
         rsp = await self.llm.Moderation.acreate(input=content)
         return rsp
+
+    @staticmethod
+    def __get_rpm(config) -> int:
```

Comment:
why __get_rpm not _get_rpm
missing input param type

Reply:
> why __get_rpm not _get_rpm missing input param type

I use __get_rpm instead of _get_rpm because this method should not be called in subclasses, and it can be accessed through self.rpm alone.

Reply:
This was replace by _get_rpm